views/djornl/djornl_node_view.json

```diff
@@ -1,19 +1,19 @@
 {
-  "writebufferIdle": 64,
-  "writebufferActive": 0,
+  "name": "djornl_node_view",
   "type": "arangosearch",
   "primarySort": [],
-  "writebufferSizeMax": 33554432,
+  "cleanupIntervalStep": 2,
   "commitIntervalMsec": 1000,
   "consolidationPolicy": {
     "type": "bytes_accum",
     "threshold": 0.1
   },
-  "globallyUniqueId": "c98232107/",
-  "cleanupIntervalStep": 2,
-  "id": "98232107",
+  "writebufferIdle": 64,
+  "writebufferActive": 0,
+  "consolidationIntervalMsec": 60000,
+  "writebufferSizeMax": 33554432,
```

Comment:
reordered lines (to match what was in the arango docs) but did not change the content of most.